packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -1283,403 +1257,96 @@ describe("toMapTree", () => {
-									: new LeafNodeStoredSchema(ValueSchema.String),
-							],
-							[brand(myMapSchema.identifier), new MapNodeStoredSchema(fieldSchema)],
-						]),
-						new Map([[fieldSchema.kind, FieldKinds.required]]),
-					);
-				}
-				it("Success", () => {
-					const schemaValidationPolicy = createSchemaAndPolicyForMapNode();
-					mapTreeFromNodeData(
-						content,
-						[myMapSchema, schemaFactory.string],
-						new MockNodeIdentifierManager(),
-						schemaValidationPolicy,
-					);
-				});
-
-				it("Failure", () => {
-					const schemaValidationPolicy = createSchemaAndPolicyForMapNode(true);
-					assert.throws(
-						() =>
-							mapTreeFromNodeData(
-								content,
-								[myMapSchema, schemaFactory.string],
-								new MockNodeIdentifierManager(),
-								schemaValidationPolicy,
-							),
-						outOfSchemaExpectedError,
-					);
-				});
-			});
-
-			describe("Array node", () => {
-				const content = ["foo"];
-				const fieldSchema = getFieldSchema(FieldKinds.required, [
-					brand(schemaFactory.string.identifier),
-				]);
-				const myArrayNodeSchema = schemaFactory.array("myArrayNode", [schemaFactory.string]);
-
-				function createSchemaAndPolicyForMapNode(invalid: boolean = false) {
-					return createSchemaAndPolicy(
-						new Map<TreeNodeSchemaIdentifier, TreeNodeStoredSchema>([
-							[
-								// An invalid stored schema will associate the string identifier to a number schema
-								brand(schemaFactory.string.identifier),
-								invalid
-									? new LeafNodeStoredSchema(ValueSchema.Number)
-									: new LeafNodeStoredSchema(ValueSchema.String),
-							],
-							[brand(myArrayNodeSchema.identifier), new MapNodeStoredSchema(fieldSchema)],
-						]),
-						new Map([[fieldSchema.kind, FieldKinds.required]]),
-					);
-				}
-				it("Success", () => {
-					const schemaValidationPolicy = createSchemaAndPolicyForMapNode();
-					mapTreeFromNodeData(
-						content,
-						[myArrayNodeSchema, schemaFactory.string],
-						new MockNodeIdentifierManager(),
-						schemaValidationPolicy,
-					);
-				});
-
-				it("Failure", () => {
-					const schemaValidationPolicy = createSchemaAndPolicyForMapNode(true);
-					assert.throws(
-						() =>
-							mapTreeFromNodeData(
-								content,
-								[myArrayNodeSchema, schemaFactory.string],
-								new MockNodeIdentifierManager(),
-								schemaValidationPolicy,
-							),
-						outOfSchemaExpectedError,
-					);
-				});
-			});
-		});
-
-		const schemaValidationPolicyForSuccess = createSchemaAndPolicy(
-			new Map([
-				[brand(schemaFactory.string.identifier), new LeafNodeStoredSchema(ValueSchema.String)],
-			]),
-			new Map(),
-		);
-		const schemaValidationPolicyForFailure = createSchemaAndPolicy(
-			new Map([
-				[
-					// Fake a stored schema that associates the string identifier to a number schema
-					brand(schemaFactory.string.identifier),
-					new LeafNodeStoredSchema(ValueSchema.Number),
-				],
-			]),
-			new Map(),
-		);
```

Comment:
This was just dead code that was unused, It has been removed.

The code above this has been moved to the prepareForInsertion tests, and the below code to various other test locations.